tasks/oracle.yml

```diff
@@ -0,0 +1,8 @@
+---
+# These tasks are only executed for Oracle Targets
+
+# Set Group Write Permissions on ORACLE_HOME/dbs for all installed ORACLE_HOMEs
+# Executed only if delphix_user is not equal to "oracle" to avoid unneeded privs
+
+- include_tasks: ohome-group-write.yml
+  when: delphix_user != 'oracle'
```

Comment:
There's no guarantee that the Oracle home owner will be named `oracle`. I'd recommend moving this check inside `ohome-group-write.yml`. You can use Ansible's `stat` module to get the owner of the `oracle` binary inside each home, then include a `when` clause to skip the mode change when the `oracle` binary's owner is the same as `delphix_user`.

Reply:
Fixed